package.json

```diff
@@ -21,8 +21,8 @@
     "format": "prettier --write .",
     "lint": "next lint",
     "plop": "plop",
-    "postinstall": "husky install && yarn certs",
-    "certs": "rm -f ./certs/VA-mozilla-combined.pem && cat node_modules/node_extra_ca_certs_mozilla_bundle/ca_bundle/ca_intermediate_root_bundle.pem ./certs/VA-Internal-S2-RCA-combined.pem > ./certs/VA-mozilla-combined.pem",
+    "postinstall": "yarn certs && husky install",
+    "certs": "rm -f ./certs/VA-mozilla-combined.pem && more node_modules/node_extra_ca_certs_mozilla_bundle/ca_bundle/ca_intermediate_root_bundle.pem > ./certs/VA-mozilla-combined.pem && more certs/VA-Internal-S2-RCA-combined.pem >> ./certs/VA-mozilla-combined.pem",
```

Comment:
Rearranged this a bit because I was having unreliable results with `cat`. Using `more` and doing it on each file individually seems to be working more reliably.